src/main/scala/dicom/DicomDeidentifyDictionary.scala

```diff
@@ -28,21 +28,26 @@ import scala.util.Success
 import scala.util.Try
 import scala.xml.XML
 
+object ActionCode extends Enumeration {
+  type ActionCode = Value
+
+  val D, Z, X, K, C, U = Value
+}
+
 case class DicomDeidElem(
     tag: Int,
     name: String,
     keyword: String,
-    action: String,
-    // todo: create Enum and use instead of Option[String]
-    retainUidsAction: Option[String] = None,
-    retainDevIdAction: Option[String] = None,
-    retainInstIdAction: Option[String] = None,
-    retainPatCharsAction: Option[String] = None,
-    retainLongFullDatesAction: Option[String] = None,
-    retainLongModifDatesAction: Option[String] = None,
-    cleanDescAction: Option[String] = None,
-    cleanStructContAction: Option[String] = None,
-    cleanGraphAction: Option[String] = None
+    action: ActionCode.Value,
+    retainUidsAction: Option[ActionCode.Value] = None,
```

Comment:
```suggestion
    retainUidsAction: Option[ActionCode.ActionCode] = None,
```

or you can use `import ActionCode.ActionCode`

```suggestion
    retainUidsAction: Option[ActionCode] = None,
```